go/exorelay/bin/lint

```diff
@@ -1,4 +1,5 @@
 #!/usr/bin/env bash
 set -e
 
+go install
```

Comment:
One of the linters needs this. See here: https://github.com/golang/go/issues/12703#issuecomment-144429128